backend/database/db_model.js

```diff
@@ -0,0 +1,58 @@
+
+const { DataTypes, Sequelize } = require('sequelize');
+const config = require('./config');
+
+const sequelize = new Sequelize(config.database, config.user, config.password, {
+  dialect: 'mssql',
```

Comment:
dialect should be mysql.... explore about this not msql both are different

Reply:
I have used mssql because I am connecting to azure database

Reply:
ok